Lynx-SwiftUI/Views/Login/LoginView.swift

```diff
@@ -216,27 +209,25 @@ struct LoginView: View {
     @ViewBuilder
     private var signLynxLogoAndSignInButtonStack: some View {
         GeometryReader { geometry in
-            VStack(alignment: .center) {
+            VStack(alignment: .center, spacing: 15) {
                 Spacer()
                 Image("LynxLogo")
                     .resizable()
                     .scaledToFit()
                     .frame(width: Constants.Logo.width)
                     .position(
                         x: geometry.size.width / 10 - Constants.Logo.xOffset,
-                        y: geometry.size.height / 1.75 - Constants.Logo.yOffset
+                        y: geometry.size.height / 2.2 - Constants.Logo.yOffset
                     )
-                    .offset(x: moveInLogo ? 0 : -200)
-                
                 if isSigningIn {
                     signInProgressView
                 } else {
                     signInWithAppleButton
-                        .offset(y: moveInApple ? 0 : 200)
```

Comment:
I think we can do this with this code? Seems a bit similar

```swift
                    VStack(spacing: 15) {
                        signInWithAppleButton
                            .offset(y: moveInApple ? 0 : 250)
                        signInWithGoogleButton
                            .offset(y: moveInGoogle ? 0 : 200)
                        signInWithFacebookButton
                            .offset(y: moveInFacebook ? 0 : 150)
                    }
```

Reply:
The offset for 10 moves the buttons up once they arrive from the animation.